### PR TITLE
Convert to pionjs

### PR DESCRIPTION
**Pull request by florianstancioiu**
Convert the `cosmoz-bottom-bar` component to pionjs

### CODE REVIEW

File: src/cosmoz-bottom-bar-next.js [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+ */
+const _layoutActions = (host, maxToolbarItems) => {
+	// eslint-disable-line max-statements
+	const elements = _getElements(host);
+	const hasActions = elements.length > 0;
+	//this._setHasActions(hasActions); // TODO: Ask about this line
+
+	if (!hasActions) {
+		// No need to render if we don't have any actions
+		// return this._setHasMenuItems(false); // TODO: Ask about this line
```
Comment (megheaiulian) [inline src/cosmoz-bottom-bar-next.js:112]:
```suggestion
		return host.toggleAttribute('has-menu-items', false);
```

File: src/cosmoz-bottom-bar-next.js [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+ *
+ * @param {*} host The current element
+ * @param {*} maxToolbarItems Maximum items for the toolbar
+ * @returns {void}
+ */
+const _layoutActions = (host, maxToolbarItems) => {
+	// eslint-disable-line max-statements
+	const elements = _getElements(host);
+	const hasActions = elements.length > 0;
+	//this._setHasActions(hasActions); // TODO: Ask about this line
```
Comment (megheaiulian) [inline src/cosmoz-bottom-bar-next.js:108]: I think we can remove this for now.

**Review by megheaiulian — CHANGES_REQUESTED** [review]
We should set the css as a constructed stylesheet using the `styleSheets` option in `component`. 
Examples can be be found in frontend. `css` and `sheet` can be imported from cosmoz-utils.

**Review by megheaiulian — CHANGES_REQUESTED** [review]
We also need to adjust exports:

@neovici/cosmoz-bottom-bar should import previous version as before.
@neovici/cosmoz-bottom-bar/next should import your new version.

Please make sure you also test this in frontend with npm link.

**Review by megheaiulian — APPROVED** [review]

**Review by cristinecula — APPROVED** [review]
Very clean work, I love it!

I noticed that it does not react to `maxToolbarItems` changes, but that's not very important right now.

**Pull request merged**